packages/web3-utils/test/fixtures/converters.ts

```diff
@@ -0,0 +1,234 @@
+
+export const bytesToHexValidData: [Bytes, HexString][] = [
+	[new Uint8Array([72]), '0x48'],
+	[new Uint8Array([72, 12]), '0x480c'],
+	[Buffer.from('0c12', 'hex'), '0x0c12'],
+	['0x9c12', '0x9c12'],
+	['0X12c6', '0x12c6'],
+];
+
+export const bytesToHexInvalidData: [any, string][] = [
+	[[9.5, 12.9], 'Invalid value given "9.5,12.9". Error: contains invalid integer values.'],
+	[[-72, 12], 'Invalid value given "-72,12". Error: contains negative values.'],
+	[[567, 10098], 'Invalid value given "567,10098". Error: contains numbers greater than 255.'],
+	[
+		[786, 12, 34, -2, 3],
+		'Invalid value given "786,12,34,-2,3". Error: contains negative values.',
+	],
+	['0x0c1g', 'Invalid value given "0x0c1g". Error: not valid hex string.'],
+	['0c1g', 'Invalid value given "0c1g". Error: not valid hex string.'],
+	['data', 'Invalid value given "data". Error: not valid hex string.'],
+	[12, 'Invalid value given "12". Error: can not parse as byte data.'],
+	[['string'], 'Invalid value given "string". Error: contains invalid integer values.'],
+	[null, 'Invalid value given "null". Error: can not parse as byte data.'],
+	[undefined, 'Invalid value given "undefined". Error: can not parse as byte data.'],
+	[{}, 'Invalid value given "[object Object]". Error: can not parse as byte data.'],
+];
+
+export const hexToBytesValidData: [HexString, Buffer][] = [
+	['0x48', Buffer.from([72])],
+	['0x3772', Buffer.from('3772', 'hex')],
+	['0x480c', Buffer.from([72, 12])],
+	['0x0c12', Buffer.from('0c12', 'hex')],
+	['0x9c12', Buffer.from('9c12', 'hex')],
+	['0X12c6', Buffer.from('12c6', 'hex')],
+];
+
+export const hexToBytesInvalidData: [any, string][] = [
+	[[9.5, 12.9], 'Invalid value given "9.5,12.9". Error: contains invalid integer values.'],
+	[[-72, 12], 'Invalid value given "-72,12". Error: contains negative values.'],
+	[[567, 10098], 'Invalid value given "567,10098". Error: contains numbers greater than 255.'],
+	[
+		[786, 12, 34, -2, 3],
+		'Invalid value given "786,12,34,-2,3". Error: contains negative values.',
+	],
+	['0x0c1g', 'Invalid value given "0x0c1g". Error: not valid hex string.'],
+	['0c1g', 'Invalid value given "0c1g". Error: not valid hex string.'],
+	['data', 'Invalid value given "data". Error: not valid hex string.'],
+	[12, 'Invalid value given "12". Error: can not parse as byte data.'],
+	[['string'], 'Invalid value given "string". Error: contains invalid integer values.'],
+	[null, 'Invalid value given "null". Error: can not parse as byte data.'],
+	[undefined, 'Invalid value given "undefined". Error: can not parse as byte data.'],
+	[{}, 'Invalid value given "[object Object]". Error: can not parse as byte data.'],
+];
+
+export const numberToHexValidData: [Numbers, HexString][] = [
+	[1, '0x1'],
+	[255, '0xff'],
+	[256, '0x100'],
+	[54, '0x36'],
+	[BigInt(12), '0xc'],
+	['768', '0x300'],
+	['-768', '-0x300'],
+	[-255, '-0xff'],
+	['0xFF0', '0xff0'],
+	['-0xa0', '-0xa0'],
```

Comment:
Add hex number test cases

```suggestion
	['-0xa0', '-0xa0'],
	[0xff, '0xff'],
	[-0xff, '-0xff'],
```